### PR TITLE
Ticket 4573: OPI for named aperture and guide positions

### PR DESCRIPTION
**Pull request by DominicOram**
### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/4573

Adds an OPI for https://github.com/ISISComputingGroup/EPICS-motionSetPoints/pull/14
Fixes pre-existing motion set point OPIs for changes made in ticket.

To test:
* Confirm all edited OPIs work

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi [hunk 1]
```diff
@@ -1,27 +1,27 @@
-<?xml version='1.0' encoding='UTF-8'?>
```
Comment (JamesKingWork) [inline base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi:1]: grep seems to suggest that this opi is not used anywhere, should we remove it?

**Review by JamesKingWork — CHANGES_REQUESTED** [review]
Something I forgot to upload a comment on in the first review: One of the acceptance criteria is that IBEX does not move when motion is disabled and the selector is greyed out when motion is disabled. This has not been fulfilled.

File: base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi (hunk 1 above)
Comment (DominicOram) [reply base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi:1]: It's listed under opi_info as `Motion Set Point (Few)` did you mean that no one is using `It's listed under opi_info as `Motion Set Point (Few)` in a synoptic/device screen on any of the instruments?

**Comment by DominicOram** [comment]
> Something I forgot to upload a comment on in the first review: One of the acceptance criteria is that IBEX does not move when motion is disabled and the selector is greyed out when motion is disabled. This has not been fulfilled.

I've greyed the button out and added a note to https://github.com/ISISComputingGroup/IBEX/issues/5588 to look at doing it server side

File: base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi (hunk 1 above)
Comment (JamesKingWork) [reply base/uk.ac.stfc.isis.ibex.opis/resources/stage/inout_motion_setpoints.opi:1]: It's been a while but I think that is what I meant, it's probably fine to leave it though

**Review by JamesKingWork — APPROVED** [review]
Looks good

**Pull request merged**